src/language-html/print/angular-control-flow-block.js

```diff
@@ -0,0 +1,67 @@
+import {
+  group,
+  hardline,
+  softline,
+  line,
+  indent,
+  join,
+  ifBreak,
+} from "../../document/builders.js";
+import { printChildren } from "./children.js";
+import settings from "./angular-control-flow-block-settings.evaluate.js";
+
+function printAngularControlFlowBlock(path, options, print) {
+  const { node } = path;
+  const setting =
+    settings.ANGULAR_CONTROL_FLOW_BLOCK_SETTINGS.get(node.name) ??
+    settings.DEFAULT_ANGULAR_CONTROL_FLOW_BLOCK_SETTINGS;
+  const docs = [];
+
+  if (
+    setting.isFollowingBlock &&
+    path.previous &&
+    path.previous.type === "block"
+  ) {
+    docs.push("} ");
+  }
```

Comment:
This part should be safer, check the previous node actually didn't print close bracket.

Reply:
@DingWeizhe Can you fix this?

**Prettier pr-15606**
[Playground link](https://deploy-preview-15606--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEABOBbADjAnsAXwB0oT0AbAZzgAJCQAaECHAS2kuVAEMAnXiAHcACnwScU3coO65OTAEa9uYANZwYAZW4Y4AGVZQ4yAGZTqi5Wo2asKwwHNkMXgFc4TahlbO3HkJSO5HAAiq4Q8Kbm-gBWlAAemkGh4ZFIZlT+AI6pcMICWBIg3JQAtEZwACZVjCAu3KzkjgDCEBgY3MjF5OS1gVAOwQCCMC6sCq7wwnC8BkZRmUwAFjAY5ADqS6zwlHZgcJri26wAbtu4XWCU8iAn7gCSUNWwmmC8rDhDT5p4wQsWICwAmo62UWC6QLg1F4J2MTEM0Jg+W4Dg6-38dl40K63AGrnIfFqQMMMHWrEqMCWyAAHAAGJi8OA5ViM5GozrpaJMGDcBRkilUpAAJiYrmoABVeRIMgDMAoqtVKnpcQ5XCi4AAxCC8DqjRw4yYQEAEAhAA)
<!-- prettier-ignore -->
```sh
--parser angular
```

**Input:**
<!-- prettier-ignore -->
```html
@empty{}

@else {}
```

**Output:**
<!-- prettier-ignore -->
```html
@empty {}

} @else {}

```

**Second Output:**
<!-- prettier-ignore -->
```html
SyntaxError: Unexpected closing block. The block may have been closed earlier. If you meant to write the } character, you should use the "&#125;" HTML entity instead. (3:1)
  1 | @empty {}
  2 |
> 3 | } @else {}
    | ^
  4 |
```